app/javascript/components/pages/__test__/SightEdit.test.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react'
+import Enzyme, { shallow } from 'enzyme'
+import Adapter from 'enzyme-adapter-react-16'
+import SightEdit from '../SightEdit'
+
+Enzyme.configure({ adapter: new Adapter()})
+
+describe('When SightEdit renders', () => {
+    const mockSight = {
+        name: "Taco Bell",
+        address1: "Bell Street",
+        city: "Taco",
+        zip_code: "11111",
+        state: "Jamaica Land",
+        country: "Jamaica",
+        phone: "222-222-2222",
+        trip_id: 1
+    }
+
+    it('displays a heading', () => {
```

Comment:
there's opportunity to also provide coverage on all the other properties of `sight` since there are inputs for each property on sight